openquake/commonlib/export/risk.py

```diff
@@ -185,6 +185,17 @@ def export_avglosses_csv(ekey, dstore):
     ('icurves-rlzs', 'csv'),
     ('rcurves-stats', 'csv'),
     ('icurves-stats', 'csv'),
+)
+def export_ebr(ekey, dstore):
+    assets = get_assets_sites(dstore)
+    outs = extract_outputs(ekey[0], dstore, dstore.export_dir, ekey[1])
+    for out in outs:
+        writers.write_csv(
+            out.path, compose_arrays(assets, out.array), fmt='%9.7E')
```

Comment:
Notice that the export functionality is the important one, view_mean_avg_losses is just an utility to show the results in a simple way, to be used only for small calculations (i.e. in the tests). There I reduced the precision a bit since it made it easier to compare the results of oq-lite and the engine and Julia. In could change the precision in view_mean_avg_losses in the future, it is very much a debugging utility still `in fieri`.